protos/consensus.proto

```diff
@@ -216,9 +216,30 @@ message ConsensusInitializeBlockResponse {
   Status status = 1;
 }
 
-// Stop adding batches to the current block and finalize it. Include
-// the given consensus data in the block. If this call is successful,
-// the consensus engine will receive it afterwards.
+// Stop adding batches to the current block and return a summary of its
```

Comment:
This command is named for its return value (the summary), but its main purpose (or at least a massive side effect) is to halt the adding of new batches to the block. Is there another name we could use? Imagine an engine writer wants to halt new batches, how would they know to use this command?

Reply:
This is a fair point, but I have struggled for a couple weeks already trying to come up with a good name. I have considered a few other options such as `complete()` (which I didn't like because it has a different meaning in schedulers) and `finalize()` (which requires renaming the existing finalize..) but this was the best one I came up with. If you have any suggestions I'd be interested in hearing them.

Reply:
I don't. `summarize` is fine, and we can change it later if we come up with something better.